test/rules/Lambda.test.ts

```diff
@@ -351,4 +353,29 @@ describe('AWS Lambda', () => {
       validateStack(stack, ruleId, TestType.VALIDATION_FAILURE);
     });
   });
+
+  describe('LambdaLogging: Ensure that Lambda functions have a corresponding Log Group', () => {
+    const ruleId = 'LambdaLogging';
+    test('Noncompliance 1', () => {
+      new CfnFunction(stack, 'rFunction', {
```

Comment:
```suggestion
      new CfnFunction(stack, 'Function', {
```

The `r` is a CloudFormation naming convention which shouldn't be used in CDK. I mistakenly followed this pattern in some of the older tests.